vnext/Microsoft.ReactNative/IReactContext.idl

```diff
@@ -13,6 +13,18 @@ import "IReactPropertyBag.idl";
 
 namespace Microsoft.ReactNative
 {
+
+  DOC_STRING(
+    "Used to represent the state of the React Native JavaScript instance")
+  enum LoadingState
+  {
+    Loading = 0,
+    Loaded = 1,
+    HasError = 2,
+    Unloaded = 3,
+    Fatal = 4
```

Comment:
is it different from HasError? maybe it'd be good to have a doc_string for what these states mean if they aren't obvious (or pick more verbose names :))
For example, if HasError, wouldn't it be Unloaded? (maybe we need a state machine showing the transitions/actions)

Reply:
Added some doc_strings